app/src/androidTest/java/org/mozilla/fenix/helpers/TestAssetHelper.kt

```diff
@@ -17,6 +17,12 @@ object TestAssetHelper {
     @Suppress("MagicNumber")
     val waitingTime: Long = TimeUnit.SECONDS.toMillis(15)
     val waitingTimeShort: Long = TimeUnit.SECONDS.toMillis(1)
+    // File name length in most operating systems is limited to 255 bytes,
+    // including file extension.
+    // This string is 251 char long though, as AS won't compile with a resource
```

Comment:
nit: expand AS to Android Studio. AS more often means "application-services" in our context :)